streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -675,27 +670,6 @@ public static void addMinAndMaxAndP99AndP90ToSensor(final Sensor sensor,
                 tags),
             new Max()
         );
-
-        sensor.add(
```

Comment:
Dropped the percentiles metric.

Reply:
Github won't let me comment on these lines, but we should remove the two percentiles-necessitated constants above (`PERCENTILES_SIZE_IN_BYTES` and `MAXIMUM_E2E_LATENCY`)

Reply:
Ah, missed those. Thanks!